code/parachain/frame/cosmwasm/src/mock.rs

```diff
@@ -496,10 +496,10 @@ impl PalletHook<Test> for MockHook {
 }
 
 impl Config for Test {
+	const MAX_FRAMES: u16 = 64;
 	type RuntimeEvent = RuntimeEvent;
 	type AccountIdExtended = AccountId;
 	type PalletId = CosmwasmPalletId;
-	type MaxFrames = MaxFrames;
```

Comment:
Why are we changing this as well?

Reply:
because that is improvable for const expressions usage and also this is super tech parameter no worth noising it in api.

Reply:
8 modify it towards using const bool operators for safety and code simplicity. no reason to use parity thing.

limit will never be dynamic i guess